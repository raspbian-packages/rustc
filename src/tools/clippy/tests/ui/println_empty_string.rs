fn main() {
    println!();
    println!("");

    match "a" {
        _ => println!(""),
    }
}
