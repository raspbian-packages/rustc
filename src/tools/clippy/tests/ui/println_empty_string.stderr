error: using `println!("")`
 --> $DIR/println_empty_string.rs:3:5
  |
3 |     println!("");
  |     ^^^^^^^^^^^^^ help: replace it with: `println!()`
  |
  = note: `-D print-with-newline` implied by `-D warnings`

error: aborting due to previous error

